utils/create_dependency_mapping.py

```diff
@@ -20,12 +20,12 @@ def topological_sort(dependencies: dict):
     while len(graph) > 0:
         # Find the nodes with 0 out-degree
         leaf_nodes = {node for node in graph if len(graph[node]) == 0}
-        # Add them to the list
-        sorting_list += list(leaf_nodes)
+        # Add them to the list as next level
+        sorting_list.append([name_mapping[node] for node in leaf_nodes])
         # Remove the leafs from the graph (and from the deps of other nodes)
         graph = {node: deps - leaf_nodes for node, deps in graph.items() if node not in leaf_nodes}
 
-    return [name_mapping[x] for x in sorting_list]
+    return sorting_list
```

Comment:
it seems to me that the return type is changed from a list of string (of modular file paths) to `list[list[str]]`.
Would be nice to explain this (i.e. the algorithm), and having a docstring about it (which is also missing on `main`).

Reply:
The caller (`find_priority_list`) to `topological_sort` still have `ordered_files` as variable, and its docstring is still `A tuple with the ordered files (list)`. If I don't make mistake, this is no longer the case. So should update there too?